lib/evss/vso_search/service.rb

```diff
@@ -31,16 +31,6 @@ def get_current_info(additional_headers = {})
       def request_headers(additional_headers)
         ssn = additional_headers['va_eauth_pnid'].presence || @user.ssn
         edipi = additional_headers['va_eauth_dodedipnid'].presence || @user.edipi
```

Comment:
Could there ever be a chance here still that edipi could be nil?

Reply:
Yes, that's definitely a possibility and we changed the logic to pull from the account instead to avoid this.